Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_TransformFollow.cs

```diff
@@ -22,45 +22,66 @@ public enum FollowMoment
         }
 
         [Tooltip("The moment at which to follow.")]
-        public FollowMoment moment = FollowMoment.OnPreRender;
+        [SerializeField]
+        private FollowMoment _moment = FollowMoment.OnPreRender;
```

Comment:
I disagree here: I think subclasses should use `moment` instead of `_moment` as the latter is just the backing field. Changing that one without going through `moment`'s setter would result in not correctly subscribing to/unsubscribing from `Camera.onPreRender`.